dashboard/app/controllers/safe_browsing_controller.rb

```diff
@@ -2,8 +2,7 @@
 
 class SafeBrowsingController < ApplicationController
   def safe_to_open
-    result = SafeBrowsing.determine_safe_to_open(params[:url])
-
+    result = params[:url].blank? ? true : SafeBrowsing.determine_safe_to_open(params[:url])
```

Comment:
nit: This simplifies to `params[:url].blank? || SafeBrowsing.determine_safe_to_open(params[:url])`